crypto/keys/keybase.go

```diff
@@ -407,7 +427,14 @@ func (kb dbKeybase) writeOfflineKey(pub tmcrypto.PubKey, name string) Info {
 
 func (kb dbKeybase) writeInfo(info Info, name string) {
 	// write the info by key
-	kb.db.SetSync(infoKey(name), writeInfo(info))
+	key := infoKey(name)
+	kb.db.SetSync(key, writeInfo(info))
+	// store a pointer to the infokey by address for fast lookup
+	kb.db.SetSync(addrKey(info.GetPubKey().Address()), key)
+}
+
+func addrKey(address tmcrypto.Address) []byte {
+	return []byte(fmt.Sprintf("%s.address", address.String()))
```

Comment:
Since we have more than one key type now, I think we should have these two suffixes as constants.